packages/apm/src/integrations/tracing.ts

```diff
@@ -175,9 +175,9 @@ export class Tracing implements Integration {
       logger.warn(`[Tracing] We added a reasonable default for you: ${defaultTracingOrigins}`);
     }
 
-    // Starting our inital pageload transaction
+    // Starting pageload transaction
     if (global.location && global.location.href) {
-      // `${global.location.href}` will be used a temp transaction name
+      // Use `${global.location.href}` as transaction name
```

Comment:
What was once a "temporary name" seems to have settled as "the name" for now.

Also not sure how useful this comment is, if the code is not clear, we could refactor to introduce a descriptive constant:

```typescript
const name = global.location.href;
Tracing.startPageloadTransaction(name, ...);
```